deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -25,8 +25,8 @@ spec:
         name: glusterfs
         env:
         - name: GB_GLFS_LRU_COUNT
-          value: 15
-        - name: TCMU_LOGDIR
+          value: "15"
+        - name: TCMU_LOG_DIR
```

Comment:
This var name has changed, please make sure it stays as `TCMU_LOGDIR`

Reply:
Sorry, missed that.